api/v1alpha1/spannerautoscaler_conversion_test.go

```diff
@@ -0,0 +1,331 @@
+package v1alpha1
+
+import (
+	"github.com/mercari/spanner-autoscaler/api/v1beta1"
+	"github.com/mercari/spanner-autoscaler/utils/pointer"
```

Comment:
Please don't write your own new package unnecessarily. Please use `k8s.io/utils/pointer`, since that is what is being used in the original function as well.